media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -98,21 +94,21 @@ <h1>Ochrana osobných údajov</h1>
 Súhlas so spracovaním osobných údajov je dotknutá osoba oprávnená kedykoľvek odvolať.
 </p>
 
-<p>
+<h3>
 <span>5.</span>
-tretie strany
```

Comment:
@martinmacko47 je to kapitola `5.` takze by mala byt rovnocenna s ostatnymi.

Reply:
Ano, je to nadpis ako kazdy iny.